.github/workflows/certificate-check.yml

```diff
@@ -0,0 +1,43 @@
+name: Check for upcoming web certificate expiry
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: 0 0 * * *
+
+permissions:
+  contents: read
+  issues: write
+
+env:
+  NODE_VERSION: lts/*
+
+jobs:
+  check-certificates:
+    runs-on: ubuntu-latest
+    outputs:
+      report: ${{ steps.check_certificates.outputs.report }}
```

Comment:
This seems unused?

Reply:
Yes, this is unused. I used https://github.com/nodejs/nodejs-dependency-vuln-assessments/blob/main/.github/workflows/check-vulns.yml as a starting point and missed tidying up this line when I realized I didn't need to spawn a second matrixed job. Thanks.

Reply:
Now removed.